pkg/engine/virtualmachinescalesets.go

```diff
@@ -441,6 +447,13 @@ func CreateAgentVMSS(cs *api.ContainerService, profile *api.AgentPoolProfile) Vi
 		vmssProperties.PlatformFaultDomainCount = to.Int32Ptr(int32(*profile.PlatformFaultDomainCount))
 	}
 
+	if profile.ProximityPlacementGroup != "" {
+		ppg := profile.ProximityPlacementGroup
+		vmssProperties.ProximityPlacementGroup = &compute.SubResource{
+			ID: to.StringPtr(ppg),
```

Comment:
Let's standardize to one usage pattern here. We can probably get rid of the temporary `ppg` variable and just use:

`ID: to.StringPtr(profile.ProximityPlacementGroup),`

Reply:
sure. will update the PR.